pkg/policy/validate.go

```diff
@@ -873,6 +873,16 @@ func ruleOnlyDealsWithResourceMetaData(rule kyverno.Rule) bool {
 }
 
 func validateResources(rule kyverno.Rule) (string, error) {
+	// if len(rule.MatchResources.RequestTypes) > 0 {
+	// 	validateRequestTypes(rule.MatchResources)
+	// }
+	if path, err := validateMatchRequestTypes(rule.MatchResources); err != nil {
```

Comment:
Need to validate `rule.match.any` and `rule.match.all`.

Reply:
Do we need to apply this check `validateMatchRequestTypes` to `rule.match.any(/all).resources` and `rule.exclude.any(/all).resources`?

Reply:
No, the check is for `rule.MatchResources.RequestTypes`.